### PR TITLE
added polynomial support

### PR DESCRIPTION
**Pull request by k2shah**

### CODE REVIEW

**Comment by k2shah** [comment]
@angeris  can you take a quick peak to make sure the jump stuff is correct?

File: src/agent.jl [hunk 1]
```diff
@@ -37,40 +44,58 @@ function set_goal_point!(a::AgentModel, position::RealVector)
...
+    order = a.order
     model = Model(ECOS.Optimizer)
 
-    @variable(model, x[1:n])
+    @variable(model, x[1:n, 1:order])
     @variable(model, dist)
 
     @objective(model, Min, dist)
-    @constraint(model, [dist; x - a.goal_point] ∈ SecondOrderCone())
+    # what does this constraint do?
```
Comment (angeris) [inline src/agent.jl]: hehe might be good to remove this one ;)

File: test/test_projectPoly.jl [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+a = AgentModel(order=3)
+set_current_position!(a, position)
+set_current_velocity!(a, vel)
+set_goal_point!(a, goal)
+set_ellipsoid!(a, "test", ob_center, obj_uncertainty)
+# calculate projection
+find_projection!(a)
+# test values
+@test isapprox(projection, a.trajectory, rtol=1e-3)
```
Comment (angeris) [inline test/test_projectPoly.jl:25]: neat !

**Review by angeris — APPROVED** [review]
Cool, overall looks good to me, just remove the random comment :D

File: src/agent.jl (hunk 1 above)
Comment (k2shah) [reply src/agent.jl]: fair enough, my Julia-fu is weak

**Pull request merged**